packages/ts-transformers/tsconfig.json

```diff
@@ -18,6 +18,7 @@
     "noUnusedLocals": true,
     "noUnusedParameters": true,
 
+    "tsBuildInfoFile": "tsconfig.tsbuildinfo",
```

Comment:
I think this is the default but explicit good

Reply:
It was reading/writing from the wrong location on my machine, and this fixed it. 🤷